src/main/kotlin/org/rust/cargo/runconfig/CargoCommandConfiguration.kt

```diff
@@ -25,6 +25,7 @@ class CargoCommandConfiguration(project: Project,
 
     var command: String = "run"
     var additionalArguments: String = ""
+    var environmentVariables: Map<String, String>? = null
```

Comment:
Thank you so much for your review.

I will fix it, but IDE tells me a few errors. So, if you’d indulge, would you accept that syntax?

I rewrote to `var environmentVariables: Map<String, String> = emptyMap()`, and 
I upgraded to snapshot from stable. Then, IntelliJ tells an error: 

> XmlSerializationException: java.lang.UnsupportedOperationException: Mutating immutable collection

Current stable plugin does not have that configuration, so IntelliJ cannot generate the CargoCommandConfiguration object.

After that, I rewrote to `var environmentVariables: Map<String, String>?`, and I rebuilt plugin. Then Kotlin Compiler tells and error: 

> Error:(28, 5) Kotlin: Property must be initialized or be abstract

That’s not allowed in kotlin syntax unlike java syntax.

I have no idea to fix it...